package/src/components/ImageGallery/hooks/useAnimatedGalleryStyle.tsx

```diff
@@ -72,12 +72,12 @@ export const useAnimatedGalleryStyle = ({
         { scaleX: -1 },
         { translateY: yScaleOffset },
         {
-          translateX: -translateX.value - xScaleOffset,
+          translateX: -xScaleOffset,
```

Comment:
Translating by `translateX.value` as well causes horizontal scrolling to happen even if you're closing the gallery, meaning it'll happen regardless of whether `IsSwiping.TRUE` is set or not. We don't need this behaviour as we want swiping to be active only when actually swiping, which is instead already handled downstream within the DOM tree. We need the initial translation to make sure that the images stay in their correct position on first load.